package/com.unity.formats.usd/Tests/Editor/ImportHelpersTests.cs

```diff
@@ -1,52 +1,139 @@
+        }
 
-        [SetUp]
-        public void CreateTestScene()
+        [Test]
+        public void ImportAsGameObjects_ImportUnderParent()
         {
-            m_scene = CreateTestUsdScene();
+            var root = new GameObject("thisIsTheRoot");
+            var scene = TestUtility.CreateTestUsdScene(ArtifactsDirectoryFullPath);
+            var usdRoot = ImportHelpers.ImportSceneAsGameObject(scene, root);
+            Assert.AreEqual(root.transform, usdRoot.transform.root, "UsdAsset is not a children of the given parent.");
+        }
+
+        [Test]
+        public void ImportAsGameObjects_SceneClosedAfterImport()
+        {
+            var scene = TestUtility.CreateTestUsdScene(ArtifactsDirectoryFullPath);
+            ImportHelpers.ImportSceneAsGameObject(scene);
+            Assert.IsNull(scene.Stage, "Scene was not closed after import.");
+        }
+
+        [Test]
+        public void ImportAsGameObjects_ImportClosedScene_LogsError()
+        {
+            var scene = TestUtility.CreateTestUsdScene(ArtifactsDirectoryFullPath);
+            scene.Close();
+            var root = ImportHelpers.ImportSceneAsGameObject(scene);
+            Assert.IsNull(root);
+            UnityEngine.TestTools.LogAssert.Expect(LogType.Error, "The USD Scene needs to be opened before being imported.");
+        }
+
+        [Test]
+        public void ImportAsGameObjects_CleanupAfterErrorUnderParent()
+        {
+            var parent = new GameObject();
+            var scenePath = TestUtility.CreateTmpUsdFile(ArtifactsDirectoryFullPath);
+            var scene = ImportHelpers.InitForOpen(scenePath);
+            scene.Close();
+
+            // This will cause an error as scene is closed
+            ImportHelpers.ImportSceneAsGameObject(scene, parent);
+
+            Assert.IsTrue(parent.transform.childCount == 0, "UsdAsset GameObject was not cleaned up after error under parent");
+            UnityEngine.TestTools.LogAssert.Expect(LogType.Error, "The USD Scene needs to be opened before being imported.");
+        }
+
+        [Test]
+        public void ImportAsGameObjects_UnderInactiveParent()
+        {
+            var parent = new GameObject();
+            var scenePath = TestUtility.CreateTmpUsdFile(ArtifactsDirectoryFullPath);
+            var scene = ImportHelpers.InitForOpen(scenePath);
+
+            parent.SetActive(false);
+            var usdObject = ImportHelpers.ImportSceneAsGameObject(scene, parent);
+
+            Assert.IsFalse(usdObject.activeInHierarchy);
+            Assert.IsTrue(usdObject.activeSelf, "The USD Scene is self-inactive when imported under an inactive parent");
         }
 
         [Test]
         public void ImportAsPrefabTest_SceneClosedAfterImport()
         {
-            ImportHelpers.ImportAsPrefab(m_scene, GetPrefabPath());
-            Assert.IsNull(m_scene.Stage, "Scene was not closed after import.");
+            var scene = TestUtility.CreateTestUsdScene(ArtifactsDirectoryFullPath);
```

Comment:
We could, but it probably isnt the best option here since there are some tests that close the scene, while in most cases we need it to be opened

We could also maybe move it to a SetUp funciton, but not all tests will use it, so its not the best solution either i think

I think it would be best to leave it as is